bin/hdfcoinc/pycbc_page_foreground

```diff
@@ -132,29 +136,44 @@ if args.output_file.endswith('.html'):
 
     mchirp, eta = mass1_mass2_to_mchirp_eta(mass1, mass2)
 
-    columns = [ifar_exc, ifar, fap_exc, fap, stat,  time1,
-               (time2-time1)*1000, mchirp, mass1, mass2, spin1z, spin2z]
+    columns = [ifar_exc, ifar, fap_exc, fap, stat,  time,
+               mchirp, mass1, mass2, spin1z, spin2z]
 
     # Not supposed to use FAP anymore, nomenclature is p-value. Should be
     # fixed consistently accross PyCBC.
     names = ['Exc. IFAR (YR)', 'Inc. IFAR (YR)', 'Exc. FAP',
              'Inc. FAP', 'Ranking Statistic', 'End Time',
-             'Time Diff. (ms)', 'mchirp', 'm1','m2', 's1z', 's2z']
+             'mchirp', 'm1','m2', 's1z', 's2z']
     format_strings = ['#.###E0', '#.###E0', '#.##E0', '#.##E0', '##.###',
-                      None, '##.##', '##.##', '##.##',
-                      '##.##', '##.##', '##.##']
+                      None, '##.##', '##.##', '##.##', '##.##', '##.##']
 
     if args.single_detector_triggers:
         single_snr = fortrigs.get_snglfile_array_dict('snr')
         single_chisq = fortrigs.get_snglfile_array_dict('chisq')
         single_chisq_dof = fortrigs.get_snglfile_array_dict('chisq_dof')
 
-        columns.extend([single_snr[ifo] for ifo in single_snr.keys()])
-        names.extend(["%s SNR" %(ifo) for ifo in single_snr.keys()])
-        format_strings.extend(["##.##" for ifo in single_snr.keys()])
-        columns.extend([single_chisq[ifo] / (single_chisq_dof[ifo] * 2 - 2) for ifo in single_chisq.keys()])
-        names.extend(["%s Red. Chisq" %(ifo) for ifo in single_chisq.keys()])
-        format_strings.extend(["##.##" for ifo in single_chisq.keys()])
+        for ifo in single_snr.keys():
+            snrs = []
+            for idx in range(len(single_snr[ifo][0])):
+                if single_snr[ifo][1][idx]:
+                    snrs.append('%.2f' % single_snr[ifo][0][idx])
+                else:
+                    snrs.append(' ')
+            columns.append(numpy.array(snrs))
+            names.append("%s SNR" %(ifo))
+            format_strings.append("##.##")
+        for ifo in single_chisq.keys():
+            chisqs = []
+            for idx in range(len(single_chisq[ifo][0])):
+                if single_chisq[ifo][1][idx]:
+                    rchisq = single_chisq[ifo][0][idx] / \
+                        (single_chisq_dof[ifo][0][idx]*2 - 2)
+                    chisqs.append('%.2f' % rchisq)
+                else:
+                    chisqs.append(' ')
+            columns.append(numpy.array(chisqs))
+            names.append("%s Red. Chisq" %(ifo))
+            format_strings.append("##.##")
```

Comment:
This will lead to order H1 SNR, L1 SNR, V1 SNR, H1 Chisq, L1 Chisq, V1 Chisq in the tables
I think that H1 SNR, H1 Chisq, L1 SNR, L1 Chisq, V1 SNR, V1 Chisq would make more sense (i.e. remove second for loop)

Reply:
No strong feeling either way on this, so I'm making this change now.